lntest/itest/lnd_test.go

```diff
@@ -6464,6 +6479,142 @@ func testInvoiceRoutingHints(net *lntest.NetworkHarness, t *harnessTest) {
 		},
 	)
 
+	listReq := &lnrpc.ListChannelsRequest{}
```

Comment:
when piggy-backing on an itest, it's nice to add a comment introducing the new part of the test, eg:
Now, we will test selection of hop hints for non-strict forwarding on private channels (etc etc)